rero_ils/modules/documents/utils_mef.py

```diff
@@ -304,7 +306,9 @@ def process(self):
             with contextlib.suppress(NoResultFound):
                 doc = Document.get_record(hit.meta.id)
                 changed = False
-                for subject in doc.get(self.name, []):
+                for subject in [
+                    res.get('entity') for res in doc.get(self.name, [])
+                ]:
```

Comment:
COSMETIC : maybe add a default value for `res.get('entity', {})`. It should prevent any error if `entity` isn't present (I know it's required into JSON schema... but 2 security systems are always better than only one)